### PR TITLE
Make sure get_fe_by_name works for simplices.

### PR DESCRIPTION
**Pull request by luca-heltai**
I had this sitting in my local deal.II installation for a while, and found out about it today...

### CODE REVIEW

File: tests/fe/get_fe_by_name_01.cc [hunk 1]
```diff
@@ -140,6 +140,19 @@ main()
...
+  gen.generate_all_codim("FE_SimplexP(2)");
+  gen.generate_all_codim("FE_SimplexDGP(1)");
+  gen.generate_all_codim("FE_SimplexDGP(2)");
+  gen.generate_all_codim("FE_SimplexP_Bubbles(2)");
+
+  // Mixed grids finite elements
+  gen.generate<3, 3>("FE_PyramidP(1)");
+  gen.generate<3, 3>("FE_PyramidDGP(1)");
+  gen.generate<3, 3>("FE_WedgeP(1)");
+  gen.generate<3, 3>("FE_WedgeDGP(1)");
```
Comment (peterrum) [inline tests/fe/get_fe_by_name_01.cc:154]: You can test quadratic wedges also.

**Review by peterrum — APPROVED** [review]

**Review by tamiko — APPROVED** [review]

File: tests/fe/get_fe_by_name_01.cc (hunk 1 above)
Comment (luca-heltai) [reply tests/fe/get_fe_by_name_01.cc:154]: The mechanism that provides the degree is the same as for all other FE, and it's tested elsewhere. Do you want me to add this here nonetheless?

**Review by kronbichler — APPROVED** [review]

**Comment by kronbichler** [comment]
Let's merge this now, I think the underlying mechanisms should be robust enough.

**Pull request merged**